### ISSUE TITLE
Massive server lag when Network Receiver is in unloaded chunk

### ISSUE THREAD

**Issue by SplenectomY** [refinedmods/refinedstorage#3468]
### Describe the bug

As title states. Having a network receiver (properly configured) in an unloaded chunk causes massive lag.

Using Spark, I profiled server ticks over 200 ms and Refined storage was taking up almost all of the CPU during those ticks. NetworkListener.onLevelTick() was highest in the stack trace.

`refinedstorage (v1.11.4)
Server thread89.18%
com.refinedmods.refinedstorage.apiimpl.network.__NetworkListener_onLevelTick_LevelTickEvent.invoke()89.18%
com.refinedmods.refinedstorage.apiimpl.network.NetworkListener.onLevelTick()89.18%
com.refinedmods.refinedstorage.apiimpl.network.node.NetworkNode.update()88.81%
com.refinedmods.refinedstorage.apiimpl.network.NetworkNodeGraph.invalidate()88.81%
com.refinedmods.refinedstorage.apiimpl.network.NetworkNodeGraph$Visitor.visit()88.81%
com.refinedmods.refinedstorage.apiimpl.network.node.NetworkTransmitterNetworkNode.visit()88.75%
com.refinedmods.refinedstorage.apiimpl.network.node.NetworkNode.visit()0.06%
com.refinedmods.refinedstorage.apiimpl.network.node.ImporterNetworkNode.update()0.18%
com.refinedmods.refinedstorage.apiimpl.network.node.ExternalStorageNetworkNode.update()0.12%
com.refinedmods.refinedstorage.apiimpl.network.node.ExporterNetworkNode.update()0.06%4ms (refinedstorage)
`

### How can we reproduce this bug or crash?

1. Build a storage system with a wireless transmitter and receiver, with the receiver not being in a loaded chunk
2. Observe server performance.

### What Minecraft version is this happening on?

Minecraft 1.19.2

### What Forge version is this happening on?

FTB Direwolf 1.19 (v.1.3.2), Forge version 41.1.0

### What Refined Storage version is this happening on?

1.11.4

### Relevant log output

```shell
refinedstorage (v1.11.4)
Server thread89.18%
com.refinedmods.refinedstorage.apiimpl.network.__NetworkListener_onLevelTick_LevelTickEvent.invoke()89.18%
com.refinedmods.refinedstorage.apiimpl.network.NetworkListener.onLevelTick()89.18%
com.refinedmods.refinedstorage.apiimpl.network.node.NetworkNode.update()88.81%
com.refinedmods.refinedstorage.apiimpl.network.NetworkNodeGraph.invalidate()88.81%
com.refinedmods.refinedstorage.apiimpl.network.NetworkNodeGraph$Visitor.visit()88.81%
com.refinedmods.refinedstorage.apiimpl.network.node.NetworkTransmitterNetworkNode.visit()88.75%
com.refinedmods.refinedstorage.apiimpl.network.node.NetworkNode.visit()0.06%
com.refinedmods.refinedstorage.apiimpl.network.node.ImporterNetworkNode.update()0.18%
com.refinedmods.refinedstorage.apiimpl.network.node.ExternalStorageNetworkNode.update()0.12%
com.refinedmods.refinedstorage.apiimpl.network.node.ExporterNetworkNode.update()0.06%4ms (refinedstorage)
```

**Comment by phit** [comment]
also seeing insane lag spikes caused by RS consistently https://spark.lucko.me/D04I8tapxM
same modpack and version

**Comment by raoulvdberge** [comment]
@SplenectomY Please link the spark report.

**Comment by SplenectomY** [comment]
@raoulvdberge https://spark.lucko.me/4ZAHKMK46Z

![image](https://user-images.githubusercontent.com/4369778/228874997-b5004fc9-4a4f-456c-bd2a-0b0f4e7becbd.png)

**Comment by raoulvdberge** [comment]
Thank you.

**Comment by raoulvdberge** [comment]
@SplenectomY @phit 
Thanks a lot for your input. I should have fixed the issues for RS 1.11.6, being released shortly.

**Comment by raoulvdberge** [comment]
Hello, this bug is now also fixed for Minecraft 1.18.2 in v1.10.6.
